xrviz/dashboard.py

```diff
@@ -123,7 +164,13 @@ def create_indexed_graph(self, *args):
         graph_opts = {'x': x,
                       'y': y,
                       'title': self.var,
-                      'cmap': 'Inferno'}
+                      'height': self.kwargs['height'],
+                      'width': self.kwargs['width'],
+                      'cmap': self.kwargs['cmap'],
+                      'colorbar': self.kwargs['colorbar']}
+        color_scale = self.kwargs['color_scale']
+        use_all_data = self.kwargs['compute min/max from all data']
```

Comment:
To be revisited later: I think right now this is actually 10/90%ile of the whole dataset if True, or min/max of first slice. I expect users to have opinions about what options should be presented here.